src/tablebuilder.ts

```diff
@@ -9,13 +9,13 @@ function formattedErrorFor(benchmark: Benchmark): string {
 }
 
 export function buildBenchmarkTable(benchmarks: Benchmark[]): string {
-    var tableString = "| Benchmark | Score | Error |\n" +
-                      "| --------- | ----- | ----- |\n";
-
+    let tableString = `| Benchmark | Score | Error |`
```

Comment:
I've never been super familiar with TS and its many different quote marks, is there an advantage to using backticks here rather than double quotes?

Reply:
Disclaimer and to be honest, I'm not familiar with TS, this must be my second code written in TS in my life 😅  

But, when fixing this issue, I read that `backticks` in TS, gives more flexibility in terms of formatting, like string templating capabilities, like in other languages, and that we could do things like this: 

```
`This is a line
    this is my second line`
```

In this case was just that this fixed it, and to maintain the code uniform for this function at least.

Reply:
Interesting, let's roll with it for now and see what happens 😋